### ISSUE TITLE
Any plan to release pre-built binaries

### ISSUE THREAD

**Issue by rulai-huajunzeng** [Microsoft/service-fabric#41]
Looks like the building takes a long time. Any plan to create pre-built binaries and put them into releases? It will be more convenient if they can be distributed as packages for popular Linux versions.

**Comment by vturecek** [comment]
Official Service Fabric release binaries for Ubuntu 16.04 are published on APT, if that's what you're looking for. See here for instructions: https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started-linux

We haven't created releases for them here on GitHub yet, but we will in the future.

**Comment by bpm-ms** [comment]
@vturecek Maybe we should point to these in the docs on this repo / clarify that people who want to use the product don't need to build from source

**Comment by vturecek** [comment]
@bpm-ms yeah at least until we get Releases set up in the repo. I'll open a PR.
